### PR TITLE
Add pretrained EfficientNetV2b1 weights

### PR DESCRIPTION
**Pull request by ianstenbit**
Scores .756 on ImageNet top-1, vs .798 paper-claimed (~95% of claimed result)

### CODE REVIEW

**Comment by tanzhenyu** [comment]
Is there any particular reason that the train accuracy (~55%) is way lower than val accuracy (~75%)?

**Comment by ianstenbit** [comment]
> Is there any particular reason that the train accuracy (~55%) is way lower than val accuracy (~75%)?

This is due to the use of RandAugment+CutMix, which makes the training dataset much harder than the validation dataset

**Comment by tanzhenyu** [comment]
> > Is there any particular reason that the train accuracy (~55%) is way lower than val accuracy (~75%)?
> 
> This is due to the use of RandAugment+CutMix, which makes the training dataset much harder than the validation dataset

That seems to suggest training loss will keep decreasing if we train more epochs, probably val loss as well?

**Review by tanzhenyu — APPROVED** [review]

**Comment by tanzhenyu** [comment]
w/ or w/o training more epochs to verify it, I think this PR is good to go

**Comment by DavidLandup0** [comment]
I've also seen training accuracy being significantly lower than validation accuracy when using RandAugment and CutMix/MixUp (~0.55 training, ~0.95 validation)

I like to think that this keeps the network aware that there's more room for improvement and won't stop updating weights to better fit the data, which is a common issue with high training accuracy. However, with more training, it might conceivably lower the validation accuracy by thinking it's more wrong than it really is? If they start diverging too much, maybe lowering the magnitude of random augmentation might help?
